### PR TITLE
Convert more classes in 'server' to Writeable. 

### PR DESCRIPTION
**Pull request by jtibshirani**
* Convert GetTask*.
* Convert RemoteInfo*.
* Convert GetFieldMappings*.
* Convert ValidateQueryRequest*.
* Convert MainResponse*.
* Convert MultiGet*.
* Convert Update*.

Relates to #34389.

### CODE REVIEW

**Comment by jtibshirani** [comment]
@elasticmachine run elasticsearch-ci/1

**Review by rjernst — APPROVED** [review]
LGTM

File: server/src/test/java/org/elasticsearch/action/support/single/instance/TransportInstanceSingleOperationActionTests.java [hunk 1]
```diff
@@ -78,13 +78,13 @@
     private TestTransportInstanceSingleOperationAction action;
 
     public static class Request extends InstanceShardOperationRequest<Request> {
-        public Request() {
-        }
+        public Request() {}
+        public Request(StreamInput in) {}
```
[the remark this answers is not in the document]
Comment (jtibshirani) [reply server/src/test/java/org/elasticsearch/action/support/single/instance/TransportInstanceSingleOperationActionTests.java]: 👍 will fix this in an immediate follow-up.

**Comment by jtibshirani** [comment]
Thanks for the review!

**Pull request merged**